### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by sarashub** [tutao/tutanota#1903]
For the Webclient:
* When selecting a menu item, folder i.a. the next tab should navigate through the respective submenu or list. E.g. when selecting the Inbox folder the next tab should navigate through the mails, when selecting the settings menu the next tab should navigate through the settings. 
* When selecting a list item the next tab should jump on the respective content, e.g. on the selected e-mail.
* The scary triangle must be mentioned when present (different envelop sender).
* Test more.

For IOS:
* Accessibility is not really working

For Android:
* Test more.

**Comment by bedhub** [comment]
Improvements for this issues are:
- NavButtons and Buttons should be triggered with Space or Enter
- Define proper Landmarks for better navigation, Banner, Navigation (top), Search, Navigation (first column), Main (second column), Region (third column) and ContentInfo (drawer menu)
- When executing action on top nav button the main column of the resulting view should be focused
- When executing action on a NavButton from the folder column the list column should be focused
- When selecting list item the third column should be focused.

**Comment by armhub** [comment]
Found issues:
When loading the login view with saved credentials, there is a visible focus border around all login buttons and a border around data privacy statement and imprint

**Comment by sarashub** [comment]
> When executing action on top nav button the main column of the resulting view should be focused

When the resulting view is already selected then nothing happens. The main column should be focused in this case as well.

**Comment by sarashub** [comment]
Found another issue:
When adding a new calendar and save it with Enter the add calendar window is not closed anymore.

**Comment by charlag** [comment]
With Orca I can still focus month switching buttons of other months. We should check if it's a problem in Orca that it ignores aria-hidden or it should really be like that.

Maybe we shouldn't even render those time intervals on desktop because there's no swipe scrolling anyway.

**Comment by charlag** [comment]
Bug with calendar caused by earlier changes to return focus to the element which had it before opening dialog. For some reason mithril dispatches event to the wrong element (new event button and not dialog "ok" button).
Needs more digging.

**Comment by charlag** [comment]
So the problem with the calendar dialog is that we listen only for the `keyup` event. And it is dispatched correctly - key is up when the element is focused. We didn't check that we got `keydown` first on this button (so `keydown` is dispatched to `KeyManager` and `keyup` is dispatched to the button which was focused when we released the key - new calendar button)

**Comment by charlag** [comment]
Fixed by 2baef6c and eb97786
